config/locales/views/users/hi.yml

```diff
@@ -0,0 +1,80 @@
+        profile_picture: "प्रोफ़ाइल फोटो"
+        choose_file: "फ़ाइल चुनें"
+        country_select_placeholder: "दर्ज न करना पसंद करें"
+        select_locale_prompt: "लोकेल चुनें"
+        subscribe_mail_checkpoint: "आपने ईमेल की सदस्यता ली?"
+        delete_account_heading: "अपने खाते को नष्ट करे"
+        delete_account_notice: "सावधानी के साथ आगे बढ़ें! इस क्रिया को पूर्ववत नहीं किया जा सकता है।"
+        delete_account: "खाता हटाएं"
+      groups:
+        title: "%{name} - आपके समुह"
+        alt_title: "Nil - आपके समुह"
+        main_heading: " आपके समूह"
+        make_new_group: "+ नया समूह बनाएं"
+        groups_user_mentor_to: "आपके द्वारा निर्देशित समूह:"
+        total_members: "कुल सदस्य: %{group_member_count}"
+        groups_user_belong_to: "आप जिन समूहों में हैं:"
+      index:
+        title: "%{name} - आपके सर्किट"
+        alt_title: "Nil - आपके सर्किट"
+        edit_details: "प्रोफाइल एडिट करें"
+        my_circuits: "मेरे बनाये सर्किट"
+        favourite_circuits: "पसंदीदा सर्किट"
+        collaborated_circuits: "सहयोग किये गए सर्किट"
+        search_circuits: "सर्किट की खोज करे..."
+        no_projects_heading_html: "आपके द्वारा बनायी कोई परियोजना नहीं है. %{simulator_new_url} परियोजना बनाए"
+        create_here: "यहां"
+        no_user_projects: "%{name} के द्वारा बनायी कोई परियोजना नहीं है। "
+        no_favourites_heading_html: "आपके द्वारा पसंद की गयी कोई परियोजना नहीं है। %{simulator_new_url} परियोजना बनाए"
+        no_user_favourites: "%{name} के द्वारा पसंद की गयी कोई परियोजना नहीं है।"
+        not_a_collaborator: "आप किसी परियोजना के सहयोगी नहीं हैं।"
+        user_not_a_collaborator: "%{name} किसी भी परियोजना के सहयोगी नहीं है।"
+      search:
+        no_results: "कोई परिणाम नहीं मिला"
+    mailer:
+      user_welcome_msg: "आपका स्वागत हे %{email}!"
+      confirm_account_description: "आप नीचे दिए गए लिंक के माध्यम से अपने खाते के ईमेल की पुष्टि कर सकते हैं:"
+      link_to_confirm_account: "मेरे खाते की पुष्टि करें"
+      email_changed_msg: "हम आपको सूचित करने के लिए आपसे संपर्क कर रहे हैं कि आपका ईमेल बदल दिया जा रहा है %{email}."
+      password_changed_msg: "हम आपको सूचित करने के लिए आपसे संपर्क कर रहे हैं कि आपका पासवर्ड बदल दिया गया है।"
+      reset_password_request_msg: "किसी ने आपका पासवर्ड बदलने के लिए एक लिंक का अनुरोध किया है। आप इसे नीचे दिए गए लिंक के माध्यम से पासवर्ड बदल सकते हैं।"
+      link_to_reset_password: "मेरा पासवर्ड बदले"
+      reset_password_description1: "यदि आपने पासवर्ड बदलने का अनुरोध नहीं किया है, तो कृपया इस ईमेल को अनदेखा करें। "
+      reset_password_description2: "आपका पासवर्ड तब तक नहीं बदलेगा जब तक आप ऊपर दिए गए लिंक को एक्सेस नहीं करते और एक नया पासवर्ड नहीं बना लेते।"
+      unlock_account_description1: "अत्यधिक संख्या में असफल साइन इन प्रयासों के कारण आपका खाता लॉक कर दिया गया है।"
+      unlock_account_description2: "अपना खाता अनलॉक करने के लिए नीचे दिए गए लिंक पर क्लिक करें:"
+      link_to_unlock_account: "मेरा खाता अनलॉक करें"
```

Comment:
instead of unlock, how about "khata khole"? Unlock is an English word and writing English words in Hindi might not be a good idea everytime. @pavanjoshi914 what do you think?

Reply:
```khata khole``` will be still like create an account. That string belong to the page which is rendered when the user account gets locked and it needs to be unlocked